tests/Rover.test.ts

```diff
@@ -0,0 +1,36 @@
+import Rover from '../src/modules/Rover';
+
+describe('Rover', () => {
```

Comment:
is the test can show the logic function L R M separated for invalid handle due to one of the sub-function.